### ISSUE TITLE
[typegraphql-prisma] Fail to generate when model does not start with capital

### ISSUE THREAD

**Issue by sns-sem** [MichalLytek/type-graphql#661]
**Describe the Bug**
If a model in your prisma schema does not start with a capital letter when running `npx prisma generate`, it will fail to generate the `resolvers/outputs/Aggregate[modelName].ts` files, instead naming them `Aggregateundefined.ts`.

For postgres as least, prisma generates all model names as lowercase by default.

**To Reproduce**
Easily replicated by have any model that starts lowercase:
```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

generator typegraphql {
  provider = "node node_modules/typegraphql-prisma/generator.js"
  output   = "../prisma/generated/type-graphql"
}

model users {
  id       String @default(dbgenerated()) @id
  username String
}

```

Generates these files:
`outputs/args/AggregateundefinedCountArgs.ts`, `outputs/Aggregateundefined.ts`

**Expected Behavior**
Properly generates the `AggregateUsersCountArgs` and `AggregateUsers.ts`

**Environment (please complete the following information):**

- OS: Windows 10
- Node v12.9.0
- TypeScript version v3.9.5

**Additional Context**
Tables in databases like postgres are, and should be, lowercase by default, this problem hinders smooth development. There is also no indication that this might be a problem anywhere, so I assume this is a bug.

**Comment by MichalLytek** [comment]
> For postgres as least, prisma generates all model names as lowercase by default.

You mean the introspection?

> If a model in your prisma schema does not start with a capital letter 

Maybe it's a prisma docs bug because they show only the models with a PascalCase names so I believe I assume that in plenty of places. Should be easy to fix tho by enforcing the transformation in file names.

**Comment by sns-sem** [comment]
> You mean the introspection?
Yes, sorry for not clarifying. It seems like with the above setup, the introspection will always generate models fully lowercase.

**Comment by MichalLytek** [comment]
Fixed in f93d241bf94618d5427188e73d94bb3f9c1def6b 🔒
